packages/dart/lib/src/hub_adapter.dart

```diff
@@ -161,6 +163,16 @@ class HubAdapter implements Hub {
         customSamplingContext: customSamplingContext,
       );
 
+  @override
+  Span startSpan(
+    String name, {
+    SpanParent parentSpan = const SpanParent.active(),
+    bool active = true,
+    Map<String, SentryAttribute>? attributes,
+  }) =>
+      Sentry.currentHub.startSpan(name,
```

Comment:
Should call `attributes` as last parameter, just like the method definition. Just o be consistent.